lite-pm/pages/components/Timeline.js

```diff
@@ -123,13 +129,15 @@ export default class Timeline extends Component {
     const [timeTicks, leftTimestamp, rightTimestamp] = this.calculateTimeTicks(
       this.state.offset
     );
+    this.props.updateParentTimelineData(this.state.offset, timeTicks, leftTimestamp, rightTimestamp)
```

Comment:
Will all Timeline objects always have this defined at all times? Vercel is giving me this error

![image](https://user-images.githubusercontent.com/17074399/109573327-db807000-7abc-11eb-91bc-f592ef48cfb5.png)

Reply:
Theres only ever one timeline object at a time and it will always get passed that function... so yeah it should always have that defined.